go/integration/pkcs11kms/key_test.go

```diff
@@ -0,0 +1,147 @@
+package pkcs11kms
+
+import (
+	"context"
+	"fmt"
+	"github.com/ThalesIgnite/crypto11"
```

Comment:
move non Go core libraries under core ones, ie add a space after line 13 and move them under there

Reply:
👍